fits/forms.py

```diff
@@ -1,7 +1,7 @@
 from django import forms
 
 # from bootstrap3_datepicker.widgets import DatePickerInput
-from .models import Workout, Goal
+from .models import Workout, Goal, Weight
```

Comment:
Weight should be connected with something, in you case it should be renamed to BodyWeight